app/uk/gov/hmrc/gform/tasklist/TaskListController.scala

```diff
@@ -79,12 +79,12 @@ class TaskListController(
         TaskListUtils.withTask(cache.formTemplate, taskSectionNumber, taskNumber) { task =>
           val sectionTitle4Ga: SectionTitle4Ga = SectionTitle4Ga(task.title.value)
 
-          if (isCompleted) {
+          if (isCompleted || cache.formTemplate.isSpecimen) {
```

Comment:
I think we shouldn't add specimen related code here. 

All we should do is to guarantee, that all tasks will stay in "NOT STARTED" state